### DISCUSSION TITLE
v4.1.alpha.8

### DISCUSSION THREAD

**Discussion by mostypc123** [mostypc123/XediX#39] [Announcements]
## What's Changed
* Enhancement: Basic UI improvements for the editor by @divyanshudhruv in https://github.com/mostypc123/XediX/pull/37
_Improves the GUI._
![image](https://github.com/user-attachments/assets/62761b75-7617-4908-9bb4-34a94f8846b2)
* Requirements.txt Generation by @mostypc123 in https://github.com/mostypc123/XediX/pull/38
_Added requirements.txt generation_
![image](https://github.com/user-attachments/assets/d470396f-b8b0-4c8a-a740-7f09a2f98b26)

## New Contributors
* @divyanshudhruv made their first contribution in https://github.com/mostypc123/XediX/pull/37

**Full Changelog**: https://github.com/mostypc123/XediX/compare/v4.1.alpha.7...v4.1.alpha.8

<hr /><em>This discussion was created from the release <a href='https://github.com/mostypc123/XediX/releases/tag/v4.1.alpha.8'>v4.1.alpha.8</a>.</em>

**Comment by divyanshudhruv** [comment]
Awesome 🔥
